src/test/scala/chiseltest/tests/ClockCrossingTest.scala

```diff
@@ -31,13 +31,13 @@ class ClockCrossingTest extends AnyFlatSpec with ChiselScalatestTester {
       io.mainOut := mainReg
     }) { c =>
       c.io.divIn.poke(0x42.U)
-      c.io.mainOut.expect(0.U)  // initial register value
+      c.io.mainOut.expect(0.U) // initial register value
```

Comment:
Not sure I agree with one space before comment, but don't feel super strongly about it

Reply:
I don’t think there are some good way to fix this.